tempest/api/workloadmgr/restore/test_exclude_bootdisk_backup.py

```diff
@@ -0,0 +1,367 @@
+                        "Search file on snapshot",
+                        tvaultconf.PASS)
+                    
+            except Exception as e:
+                reporting.add_test_step(
+                            "Search file on snapshot", tvaultconf.FAIL)
+
+            ### Selective restore ###
+            rest_details = {}
+            rest_details['rest_type'] = 'selective'
+            rest_details['network_id'] = CONF.network.internal_network_id
+            rest_details['subnet_id'] = self.get_subnet_id(
+                CONF.network.internal_network_id)
+            volumeslist = []
+            rest_details['instances'] = {vm_id: volumeslist}
+
+            payload = self.create_restore_json(rest_details)
+            # Trigger selective restore
+            restore_id_1 = self.snapshot_selective_restore(
+                wid,
+                snapshot_id,
+                restore_name=tvaultconf.restore_name,
+                restore_cleanup=True,
+                instance_details=payload['instance_details'],
+                network_details=payload['network_details'])
+            self.wait_for_snapshot_tobe_available(wid, snapshot_id)
+            if(self.getRestoreStatus(wid, snapshot_id, restore_id_1) == "available"):
+                reporting.add_test_step("Selective restore", tvaultconf.PASS)
+            else:
+                reporting.add_test_step("Selective restore", tvaultconf.FAIL)
+                raise Exception("Selective restore failed")
+
+            # Fetch instance details after restore
+            vm_list = self.get_restored_vm_list(restore_id_1)
+            LOG.debug("Restored vm(selective) ID : " + str(vm_list))
+            time.sleep(60)
+            floating_ip_2 = self.assign_floating_ips(vm_list[0], True)
+            LOG.debug(
+                "Floating ip assigned to selective restore vm -> " +
+                str(floating_ip_2))
+
+            # connect to remote machine to check file on restored vm instance. 
+            ssh = self.SshRemoteMachineConnectionWithRSAKey(str(floating_ip_2))
+            self.install_qemu(ssh)
+
+            cmd = 'sudo ls -lrth /opt/File_1'
+            stdin, stdout, stderr = ssh.exec_command(cmd)
+
+            time.sleep(10)
+
+            output = stdout.read().decode('utf-8').strip()
+            error = stderr.read().decode('utf-8').strip()
+
+            LOG.debug("Output: " + str(output))
+            LOG.debug("Error: " + str(error))
+
+            if error:
+                err_msg = ["No such file or directory"]
+                result = all(x in error for x in err_msg)
+                if (result):
+                    reporting.add_test_step("Search files on restored vm instance",
+                                            tvaultconf.PASS)
+                    reporting.set_test_script_status(tvaultconf.PASS)
+                    tests[1][1] = 1
+                else:
+                    reporting.add_test_step("Different error occurred.",
```

Comment:
Please reword this to reflect correct step failure. You can use same text from line #222.